### PR TITLE
Feature/refactor question

### PR DESCRIPTION
**Pull request by svenjens**

### CODE REVIEW

File: app/components/Questionnaire/Question.js [hunk 1]
```diff
@@ -15,21 +15,23 @@ const hasKeys = obj =>
   Object.entries(obj).map(([key, value]) => [key, value]).length;
 
 const Question = ({
-  questionId,
+  question: {
+    id: questionId,
+    vraagTekst: heading,
+    antwoordOpties: answers,
+    media,
+    toelichting: paragraph,
```
Comment (robinpiets) [inline app/components/Questionnaire/Question.js]: Let's rename `paragraph` to `explanation`

File: app/containers/AddressInput/index.js [hunk 2]
```diff
@@ -171,10 +171,31 @@ class AddressInput extends React.Component {
 
     const inputError = this.validateAddressInput();
 
+    const question = {
+      vraagTekst: 'Waar wilt u uw aanbouw maken?',
```
Comment (robinpiets) [inline app/containers/AddressInput/index.js]: Let's add an ID `location` here

File: app/containers/AddressInput/index.js [hunk 3]
```diff
@@ -171,10 +171,31 @@ class AddressInput extends React.Component {
...
+    };
+
+    const questionVerify = {
+      id: 'addressVerify',
```
Comment (robinpiets) [inline app/containers/AddressInput/index.js]: Let's rename this to `locationVerify`

File: app/components/Questionnaire/Question.js (hunk 1 above)
Comment (svenjens) [reply app/components/Questionnaire/Question.js]: 🙌

File: app/components/Questionnaire/Question.js [hunk 4]
```diff
@@ -69,7 +71,7 @@ const Question = ({
   return (
     <Form className={className} onSubmit={handleSubmit(onSubmit)} data-id={questionId} {...otherProps}>
       {heading && <Heading $as={headingAs}>{heading}</Heading>}
-      {paragraph && <ReactMarkdown source={paragraph} renderers={{ paragraph: Paragraph }} linkTarget="_blank" />}
+      {explanation && <ReactMarkdown source={explanation} renderers={{ paragraph: Paragraph }} linkTarget="_blank" />}
```
Comment (robinpiets) [inline app/components/Questionnaire/Question.js]: We need to think of the naming for these props..

**Pull request merged**